tests/foreman/api/test_contentview.py

```diff
@@ -12,6 +12,8 @@
 
 """
 
+# This file fully passed in snaps 72-75, let's start here.
```

Comment:
Just for awareness, If you would like to run an upstream PRT, the PRT comment can be added directly to the Jenkins job while build with parameters, without requiring robottelo PRs, and you can also post results on any GH PRs where you wish, such as here on upstream Katello PR